backend/app/api/routes/guardrails.py

```
@@ -189,4 +182,4 @@ def add_validator_logs(guard: Guard, request_log_id: UUID, validator_log_crud: V
             outcome=ValidatorOutcome(result.outcome.upper()),
         )
 
-        validator_log_crud.create(log=validator_log)
+        validator_log_crud.create(log=validator_log)
```

Ruff (0.14.14)

No newline at end of file

Add trailing newline

(W292)